content/chat.adoc

```diff
@@ -81,10 +81,10 @@ We currently offer the following cloak levels:
 
 To obtain a cloak:
 
-* ensure you have a *registered user* on Freenode
+* ensure you have a *registered user* on Libera Chat
 * ensure you are already recognized as a developer (have +vV voice privilege)
-* email jieryn@gmail.com with those details
-* monitor your memoserv notices for updates from Freenode staff requesting confirmation of cloak request
+* Contact a https://www.jenkins.io/sigs/platform/[Platform SIG] group member, who will request a cloak on your behalf in `#libera-community`
```

Comment:
i don't think platform sig sounds right? I would say jenkins-infra or board

Reply:
> just for the record, https://gitter.im/jenkinsci/platform-sig group members

> i don't think platform sig sounds right? I would say jenkins-infra or board

OK, I will change this to "Jenkins infra" then. Also happy to change it to "Jenkins board" if that is requested.